apricot/local/service.go

```diff
@@ -477,6 +477,23 @@ func (s *Service) SetRuntimeEntry(component string, key string, value string) er
 	}
 }
 
+func (s *Service) GetRuntimeEntries(component string) (map[string]string, error) {
+	s.logMethod()
+
+	if keys, err := s.ListRuntimeEntries(component); err == nil {
+		entries := make(map[string]string)
+		for _, key := range keys {
+			if entry, err := s.GetRuntimeEntry(component, key); err == nil {
```

Comment:
This err is still unhandled.

In this kind of situation, elsewhere in AliECS we use the `go-multierror` package. Take a look for example at https://github.com/AliceO2Group/Control/blob/master/core/workflow/aggregatorrole.go#L171

Reply:
How exactly should it be handled? I thought that not adding entry that caused the error is enough. But according to multierror package you would prefer to return error from the function even if only one of X entries returns error and other are ok?

Reply:
Yes, that's the idea: if any number of entries were not returned because of an error, the errors should be packaged into a single error and returned. We still return the values we got, but we also inform the caller that something is amiss.

Reply:
something like this?